js/logo.js

```diff
@@ -4852,17 +4848,32 @@ function Logo(pitchtimematrix, pitchdrummatrix, rhythmruler,
         } else if (logo.blocks.blockList[blk].isArgBlock() || logo.blocks.blockList[blk].isArgClamp()) {
             switch (logo.blocks.blockList[blk].name) {
             case 'loudness':
-                try {  // DEBUGGING P5 MIC
-                    if (!logo.mic.enabled) {
-                        logo.mic.start();
+                try {  
+                    var analyser = new Tone.Analyser({
+                        "type" : "waveform",
+                        "size" : 256
+                    });
+
+                    logo.mic.connect(analyser);
+                    var values = analyser.analyse();
+                    var sum = 0;
+                    for(var k=0; k<1024; k++){
+                        var m = 0;
```

Comment:
Do we really need to use m as an intermediate variable? Why not just sum += (values[k] * values[k]);  ?